src/app/modules/core/components/sidebar/sidebar.component.ts

```diff
@@ -167,21 +134,36 @@ export class SidebarComponent {
     try {               
         switch (pType) {
             case ProcedureType.MAPLAYERS:
+              if(this.SelectedProcedureType === 0) {
```

Comment:
keep accordion panel open  at all times to guide the user

Reply:
Created a previousProcedureType to always revert to the previously opened panel when user closes a panel - except in the case that map layers was the previously opened panel in which case the active panel remains open.